platforms/jvm/language-jvm/src/main/java/org/gradle/api/tasks/compile/BaseForkOptions.java

```diff
@@ -96,7 +96,6 @@ public List<String> getJvmArgs() {
     public void setJvmArgs(@Nullable List<String> jvmArgs) {
         this.jvmArgs = jvmArgs == null ? null : jvmArgs.stream()
             .filter(Objects::nonNull)
-            .map(String::trim)
             .filter(string -> !string.isEmpty())
```

Comment:
Can this be changed to a `org.apache.commons.lang3.StringUtils#isBlank` call instead? The intention of this code (introduced to fix https://github.com/gradle/gradle/issues/11490) is to filter out bogus empty/blank arguments. This would still let your property setting work without also breaking situations similar to the original issue.

Reply:
I changed this as suggested and adjusted the
`BaseForkOptionsTest.JVM options are filtered properly even with bad input`
test to reflect the expected behavior.